website_docs/getting-started.md

```diff
@@ -3,552 +3,897 @@ title: "Getting Started"
 
-To start building the application, make a new directory named `fib` to house our Fibonacci project. Next, add the following to a new file named `fib.go` in that directory.
+To begin, you'll want to install OpenTelemetry and the `net/http`
+instrumentation package:
 
-```go
-package main
-
-// Fibonacci returns the n-th fibonacci number.
-func Fibonacci(n uint) (uint64, error) {
-	if n <= 1 {
-		return uint64(n), nil
-	}
-
-	var n2, n1 uint64 = 0, 1
-	for i := uint(2); i < n; i++ {
-		n2, n1 = n1, n1+n2
-	}
-
-	return n2 + n1, nil
-}
 ```
+go get go.opentelemetry.io/otel \
+  go.opentelemetry.io/otel/trace \
+  go.opentelemetry.io/otel/sdk \
+  go.opentelemetry.io/otel/exporters/otlp/otlptrace \
+  go.opentelemetry.io/otel/exporters/stdout/stdouttrace \
+  go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp \
+  go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp
+```
+
+## Create the sample HTTP Server
 
-With your core logic added, you can now build your application around it. Add a new `app.go` file with the following application logic.
+In a new project, create or edit your `main.go` file to be the following:
 
 ```go
 package main
 
 import (
-	"context"
 	"fmt"
-	"io"
 	"log"
+	"math/rand"
+	"net/http"
 )
 
-// App is a Fibonacci computation application.
-type App struct {
-	r io.Reader
-	l *log.Logger
-}
-
-// NewApp returns a new App.
-func NewApp(r io.Reader, l *log.Logger) *App {
-	return &App{r: r, l: l}
+// Implement an HTTP Handler func to be instrumented later
+func handleRollDice(w http.ResponseWriter, r *http.Request) {
+	value := rand.Intn(6) + 1
+	fmt.Fprintf(w, "%d", value)
 }
 
-// Run starts polling users for Fibonacci number requests and writes results.
-func (a *App) Run(ctx context.Context) error {
-	for {
-		n, err := a.Poll(ctx)
-		if err != nil {
-			return err
-		}
-
-		a.Write(ctx, n)
-	}
-}
-
-// Poll asks a user for input and returns the request.
-func (a *App) Poll(ctx context.Context) (uint, error) {
-	a.l.Print("What Fibonacci number would you like to know: ")
-
-	var n uint
-	_, err := fmt.Fscanf(a.r, "%d\n", &n)
-	return n, err
-}
+func main() {
+        http.HandleFunc("/rolldice", handleRollDice)
 
-// Write writes the n-th Fibonacci number back to the user.
-func (a *App) Write(ctx context.Context, n uint) {
-	f, err := Fibonacci(n)
+	err := http.ListenAndServe(":8080", nil)
 	if err != nil {
-		a.l.Printf("Fibonacci(%d): %v\n", n, err)
-	} else {
-		a.l.Printf("Fibonacci(%d) = %d\n", n, f)
+		log.Fatal(err)
 	}
 }
 ```
 
-With your application fully composed, you need a `main()` function to actually run the application. In a new `main.go` file add the following run logic.
+When run, this will launch an HTTP server that does a "dice roll" whenever the
+`/rolldice` route is accessed.
 
-```go
-package main
+## Add HTTP Server instrumentation
 
-import (
-	"context"
-	"log"
-	"os"
-	"os/signal"
-)
+[Instrumentation libraries]({{< relref "libraries" >}}) are used to create
```

Comment:
The `relref` link pattern is what we're using in the docs site because it gives hugo the ability to check links at build time. I can change this back to normal markdown links if preferred, but it does have a benefit of letting us know if something is broken each time we pull in docs changes via a submodule update.

Reply:
@cartermp - FYI, moving forward, for any markdown link that doesn't go into the website specification can use a regular path as opposed to a `relref`.

Reply:
I'd actually prefer to not use `relref`, so I can switch these back.